impl/vulkan/pipeline/BrdfmapComputer.cpp

```diff
@@ -7,6 +7,7 @@ import :vulkan.pipeline.BrdfmapComputer;
 
 import std;
 
+#define COMPILED_SHADER_DIR "shader"  
```

Comment:
Isn't it already defined by `CMakeLists.txt`?

https://github.com/stripe2933/vk-gltf-viewer/blob/713156eefed5955fa5ebd5c2f77463854bb42bce/CMakeLists.txt#L194

Reply:
> Isn't it already defined by `CMakeLists.txt`?
> 
> https://github.com/stripe2933/vk-gltf-viewer/blob/713156eefed5955fa5ebd5c2f77463854bb42bce/CMakeLists.txt#L194

yeah sorry for not being careful temporarily put it there because i was getting error there while calling binary as ./build/vk-gltf-viewer